auto_editor/Fix1003Rule.py

```diff
@@ -28,11 +28,9 @@ def run_rule(self, project: StructuredProjectSource,
         tmp_dict = project.paths_to_lines
```

Comment:
remove “print（“----------------------------”）” or left some print info below

Reply:
good point, some output is probably helpful because it can take some time. I added a message